modules/phase_field/test/tests/initial_conditions/PolycrystalVoronoi_fromfile.i

```diff
@@ -0,0 +1,60 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 40
+  ny = 40
+  nz = 0
+  xmax = 250
+  ymax = 250
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[GlobalParams]
+  op_num = 7
+  var_name_base = gr
+[]
+
+[Variables]
+  [./PolycrystalVariables]
+  [../]
+[]
+
+[ICs]
+  [./PolycrystalICs]
+    [./PolycrystalColoringIC]
+      polycrystal_ic_uo = voronoi
+    [../]
+  [../]
+[]
+
+[UserObjects]
+  [./voronoi]
+    type = PolycrystalVoronoi
+    coloring_algorithm = jp
+    grain_num = 27
```

Comment:
This parameter should now be made optional because it is determined from the file. It' confusing to have to put it in and worse, the value is actually ignored.